be/src/storage/lake/lake_local_persistent_index.cpp

```diff
@@ -293,17 +294,18 @@ Status LakeLocalPersistentIndex::load_from_lake_tablet(starrocks::lake::Tablet*
         return status;
     }
 
-    RETURN_IF_ERROR(_delete_expired_index_file(_version, _l1_version,
-                                               _l2_versions.size() > 0 ? _l2_versions[0] : EditVersion()));
+    RETURN_IF_ERROR(_delete_expired_index_file(
```

Comment:
How much disk occupation improvement after this pr?

Reply:
This PR only takes affect on minor compaction policy, it will save almost 50% disk usage compared to before.